go-controller/pkg/ovn/policy.go

```diff
@@ -290,18 +327,16 @@ func (oc *Controller) createMulticastAllowPolicy(ns string, nsInfo *namespaceInf
 	return nil
 }
 
-func deleteMulticastACLs(ns, portGroupHash string, nsInfo *namespaceInfo) error {
-	err := deleteACLPortGroup(portGroupHash, fromLport,
-		defaultMcastAllowPriority, "ip4.mcast", "allow",
-		knet.PolicyTypeEgress)
+func deleteMulticastACLs(ns, portGroupHash string) error {
+	err := deleteACLPortGroup(portGroupHash, fromLport, defaultMcastAllowPriority,
+		"ip4.mcast", "allow", knet.PolicyTypeEgress)
```

Comment:
Is seems like the code should just log an error on line 334, and proceed on to try to attempt to the next call to deleteACLPortGroup().. else, these would never be cleaned up, right?

Reply:
Well... If the 1st Port Group fails to delete and then the second one succeeded we would have no idea anything was orphaned unless were actively monitoring the logs... I would rather fail on any deletion which would point the user to directly to what's going on